parser_library/test/parsing/string_test.cpp

```diff
@@ -93,106 +179,666 @@ TEST_P(parser_string_fixture, basic)
     EXPECT_EQ(get_var_value<C_t>(a.hlasm_ctx(), "PAR"), GetParam().expected);
 }
 
-TEST(parser, no_ending_apostrophe)
+TEST(parser, incomplete_string)
 {
     std::string input = R"(
+ GBLC &PAR
  MACRO
  MAC &VAR
+ GBLC &PAR
+&PAR SETC '&VAR'
  MEND
  
- MAC "N'SYM)";
```

Comment:
where was the `"N'SYM` and `"L'SYM' STH` moved to?

Reply:
It was moved further below where other attributes are tested as well. 
See `TEST_P(parser_data_attribute_fixture, no_ending_apostrophe)`